src/RecognitionClient.cpp

```diff
@@ -81,17 +82,24 @@ void RecognitionClient::connect(const Configuration& configuration) {
             return;
         }
 
-        speechcenter::recognizer::v1::RecognitionStreamingRequest audioRequest = buildAudioRequest(configuration);
         INFO("Sending audio...");
-        streamFail = !stream->Write(audioRequest);
-        if(streamFail) {
-            auto status = stream->Finish();
-            ERROR("{} ({}: {})",  status.error_message(), status.error_code(), status.error_details());
-            return;
+        int requestCount = 0;
```

Comment:
maybe to initialize the 
`int requestCount = 1;`
to 1 instead of 0, just for the logs, and in line 102 to subtract one
`INFO("All audio sent in {} requests.", (requestCount -1) );`

Reply:
I do not really liked it, because if the for is not executed because requests are not generated, the requestsCount would be one anyway.

Reply:
the requestCount shouldn't be one, because if we initialize to 1 we need to subtract also from:
`INFO("All audio sent in {} requests.", (requestCount -1) );`

Reply:
Then you just `print` the right number but the variable content is wrong.  I really do not see the benefits of doing it this way.

Reply:
I move the requestsCount one line up so that the print for bytes sent is not zero.